graphql/integration_atomic_test_util.go

```diff
@@ -690,20 +688,3 @@ func addSubnets(t *testing.T) {
 func clearSubnets(t *testing.T) {
 	evergreen.GetEnvironment().Settings().Providers.AWS.Subnets = []evergreen.Subnet{}
 }
-
-func additionalChecks(t *testing.T) {
-	var checks = map[string]func(*testing.T){
-		// note these 2 are only the same because the same project ID is used
-		"TestAtomicGQLQueries/abortTask/commit-queue-dequeue.graphql":            checkCommitQueueDequeued,
-		"TestAtomicGQLQueries/unschedulePatchTasks/commit-queue-dequeue.graphql": checkCommitQueueDequeued,
```

Comment:
Only somewhat related, but is [this file](https://github.com/evergreen-ci/evergreen/blob/ddb5a15fe74794739642ec6e9aa02d46c56c29c2/graphql/tests/mutation/unscheduleVersionTasks/queries/commit_queue_dequeue.graphql) used anymore? I can't tell if it matters but the `data.json` has some stuff in it that looks like legacy commit queue logic.

I'm also fine if we plan to defer test data updates to a later PR.

Reply:
Whoa good find